docs: Move N4A FAQ content to docs (#537)
nginx/documentation: content/nginxaas-azure/billing/overview.md

- concept
---

> NGINXaaS for Azure is deployed into your Azure subscription. Your NGINXaaS deployment resource is visible within your subscription, while the underlying compute infrastructure is managed by F5 and not exposed in your Azure resource list. This separation enables simplified management while maintaining visibility over your service instance.

Reviewer: A couple of things here:
- Maybe don't use the word `Instance` here and instead stick with `deployment`. Instance becomes overloaded with what it means in N1C vs here and it kind of leans towards what the underlying infrastructure might look like which may or may not be what a user thinks of when they see the word `instance`.
- Adding on to the above, skip calling out `compute` infrastructure and try keeping it generalized. It is a mix of things and we should stick with the higher-level entity -> `NGINXaaS deployment` which comprises of `managed infrastructure`.
```suggestion
NGINXaaS for Azure is deployed into your Azure subscription. Your NGINXaaS deployment resource is visible within your subscription, while the underlying infrastructure is managed by F5 and is abstracted away from you. This separation enables simplified management while maintaining visibility over your service instance.
```

{{< note >}}The management traffic for NGINX instances is billed as a `Virtual Network Peering - Intra-Region Egress` charge. This charge includes the data for shipping metrics and logs. The cost for shipping metrics data is approximately $0.03/month. If you enable NGINX logging the cost increases by roughly $0.005 per GB of logs NGINX generates. To estimate this, multiply the number of requests by the average log line size of the access_log format you have configured.
{{< /note >}}

> ## Usage Data Reporting​

Reviewer: While this is in the context of billing, reading this header and the first sentence here makes it sound like how NGINXaaS is utilizing customer usage data. You may want to look at clarifying the headers and the underlying section here to reflect how a user can view their _billing data_.

Author: Not sure I understand what your ask is here.

This is content copied from the FAQ.

Should it go somewhere else or can it be removed?

If we want to address something different in this document, that's outside of the scope of the PR and we can work on it separately.

Author: ```suggestion
## Review billing data
```

Reviewer: the newly suggested header works 👍 and is clearer.

> NGINXaaS usage data is reported directly to NGINXaaS. This data is utilized for billing purposes and is reflected in your Azure subscription's cost analysis. You can view detailed usage information in the Azure Portal under "Cost Management + Billing".​

Reviewer: ```suggestion
NGINXaaS billing data is reported per deployment and can be viewed in the Azure Portal under "Cost Management + Billing".​
```

Sounds nicer?
